.goreleaser.yml

```diff
@@ -37,30 +43,30 @@ snapshot:
 changelog:
   sort: asc
 
-dockers:
-  - goos: linux
-    goarch: amd64
-    ids:
-      - reinforcer
-    image_templates:
-      - 'clear-street/reinforcer:{{ .Tag }}'
-      - 'clear-street/reinforcer:v{{ .Major }}'
-      - 'clear-street/reinforcer:v{{ .Major }}.{{ .Minor }}'
-      - 'clear-street/reinforcer:latest'
-    build_flag_templates:
-      - "--pull"
-      - "--label=org.opencontainers.image.created={{.Date}}"
-      - "--label=org.opencontainers.image.name={{.ProjectName}}"
-      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
-      - "--label=org.opencontainers.image.version={{.Version}}"
-      - "--label=org.opencontainers.image.source={{.GitURL}}"
-
-brews:
-  - homepage: https://github.com/clear-street/reinforcer
-    description: "Codegen tool for easy adoption of resiliency patterns in Go"
-    tap:
-      owner: clear-street
-      name: homebrew-reinforcer
-    folder: Formula
-    test: |
-      system "#{bin}/reinforcer --version"
+#dockers:
+#  - goos: linux
```

Comment:
same q

Reply:
Left it in case we wanted to resume publishing docker builds in the future